grammar.js

```diff
@@ -21,11 +21,7 @@ module.exports = grammar({
         comment: (_) => token(seq("#", /.*/)),
 
         // LIST http verb is arbitrary and required to use vaultproject
-        method: ($) =>
-            choice(
-                /(OPTIONS|GET|HEAD|POST|PUT|DELETE|TRACE|CONNECT|PATCH|LIST)/,
-                $.const_spec,
```

Comment:
I couldn't find any tests referencing this and it was interfering with parsing the optional response, so I ended up removing it.

Reply:
I found it was introduced in https://github.com/rest-nvim/tree-sitter-http/pull/10, not sure what's its purpose

Reply:
In theory it should allow even unlisted methods, but it is something that is not supported by cURL either. I think it is safe to remove it, in case it is needed again, we can add it again